app/src/main/res/layout/fragment_detailed_item.xml

```diff
@@ -225,9 +225,24 @@
                 android:layout_width="match_parent"
                 android:layout_height="wrap_content"
                 android:layout_gravity="fill_horizontal|top"
-                android:contentDescription="@string/share"
                 android:backgroundTint="@color/themeColor"
+                android:layout_marginTop="16dp"
+                android:contentDescription="@string/share"
                 app:srcCompat="?attr/actionModeShareDrawable" />
+
+            <Space
+                android:layout_width="10dp"
+                android:layout_height="wrap_content" />
+
+            <com.google.android.material.floatingactionbutton.FloatingActionButton
+                android:id="@+id/locateButton"
+                android:layout_width="match_parent"
+                android:layout_height="wrap_content"
+                android:layout_gravity="end|bottom"
+                android:backgroundTint="@color/themeColor"
+                android:layout_marginTop="16dp"
+                android:contentDescription="@string/locate_item"
+                app:srcCompat="@drawable/baseline_search_black_24dp" />
```

Comment:
Could you just make the icon white?

Reply:
![image](https://user-images.githubusercontent.com/22821947/118825308-db715e80-b8ba-11eb-804f-433ba76a3267.png)
Replaced with this icon

Reply:
Perfect